rpc/params/firewall.go

```diff
@@ -58,5 +58,5 @@ func (v KnownServiceValue) Validate() error {
 	case SSHRule, JujuControllerRule, JujuApplicationOfferRule:
 		return nil
 	}
-	return errors.NotValidf("known service %q", v)
```

Comment:
where did this change come from?

Reply:
I managed to pull it out of my own records, apparently it's coming from this one (from September 2022) https://github.com/juju/juju/pull/14608, looks like we didn't like the resulting error message https://github.com/juju/juju/pull/14608#discussion_r970814743